### DISCUSSION TITLE
Student Benifits Issues

### DISCUSSION THREAD

**Discussion by MiloKesteloot** [community/community#135267] [GitHub Education]
### Hi everyone,

Hello! I applied for the student benifits and was accepted 8 days ago  but have not receved any benifits. Is there anything more I should do? Thanks :)

**Comment by soupaulogomes** [comment]
I've the same problem. Accepted very fast but don't receive my benefits

**Comment by davevad93** [comment]
Hi @cardy64 ,

You don't have to do anything more, just wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @cardy64 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
